### PR TITLE
add cilium e2e

### PR DESCRIPTION
**Pull request by lut777**
As title.
In e2e, kube-proxy is removed and cilium is introduced in but the svc dnat is still available.

### CODE REVIEW

File: .github/workflows/build-x86-image.yaml [hunk 1]
```diff
@@ -936,6 +936,60 @@ jobs:
...
+        with:
+          go-version: ^1.17
+        id: go
+
+      - name: Run E2E
+        run: |
+          go install github.com/onsi/ginkgo/ginkgo@latest
+          sudo kubectl cluster-info
+          sudo chmod -R 777 /home/runner/.kube/
+          make e2e-ovn-ebpf
```
Comment (oilbeater) [inline .github/workflows/build-x86-image.yaml:986]: Should run the basic e2e too

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: .github/workflows/build-x86-image.yaml (hunk 1 above)
Comment (lut777) [reply .github/workflows/build-x86-image.yaml:986]: can't. basic e2e would check the status of kube-proxy which is already deleted in this scenario.

File: .github/workflows/build-x86-image.yaml [hunk 2]
```diff
@@ -936,6 +936,64 @@ jobs:
...
+        id: go
+
+      - name: Run E2E
+        run: |
+          go install github.com/onsi/ginkgo/ginkgo@latest
+          sudo kubectl cluster-info
+          sudo cp -r /root/.kube/ /home/runner/.kube/
+          sudo chmod -R 777 /home/runner/.kube/
+          docker exec kube-ovn-control-plane bash -c "ls -al /etc/cni/net.d/"
+          sleep 30
```
Comment (oilbeater) [inline .github/workflows/build-x86-image.yaml:989]: the sleep can be removed

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: .github/workflows/build-x86-image.yaml (hunk 2 above)
Comment (lut777) [reply .github/workflows/build-x86-image.yaml]: kubectl  rollout status -n kube-system ds cilium
Above command is added and replaced the "sleep" to ensure cilium pods are properly started.

File: test/e2e-ebpf/e2e_ebpf_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
+package e2e_ebpf
+
+import (
+	"context"
+	_ "embed"
```
Comment (zhangzujian) [inline test/e2e-ebpf/e2e_ebpf_suite_test.go:5]: Package `embed` shoud be removed from the import list.

Comment (lut777) [reply test/e2e-ebpf/e2e_ebpf_suite_test.go:5]: deleted

**Review by oilbeater — APPROVED** [review]

**Pull request merged**